dotnet-efcore/VirtualKeywordInEFCore/Tests/VirtualKeywordInEFCoreTests.cs

```diff
@@ -0,0 +1,61 @@
+using Microsoft.EntityFrameworkCore;
+using VirtualKeywordInEFCore.Models;
+
+namespace Tests
+{
+    public class VirtualKeywordInEFCoreTests
+    {
+        private readonly DataContextWithoutLazyLoading _context;
```

Comment:
These can be instantiated where they're defined, and then taken out of the constructor.

```suggestion
        private readonly DataContextWithoutLazyLoading _context = new();
```